src/js/bootstrap-wysiwyg.js

```diff
@@ -0,0 +1,200 @@
+/* http://github.com/mindmup/bootstrap-wysiwyg */
```

Comment:
Please use the latest version of bootstrap-wysiwyg with script tag.
You can find the latest version [here](https://cdnjs.com/libraries/bootstrap3-wysiwyg).